enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/RegisterAllocation.scala

```diff
@@ -36,21 +35,16 @@ object RegisterAllocation {
     def allocate(lp: LogicalPlan, nullable: Boolean, argument: Option[PipelineInformation]): PipelineInformation = lp match {
       case Aggregation(source, groupingExpressions, aggregationExpressions) =>
         val oldPipeline = allocate(source, nullable, argument)
-        val newPipeline = PipelineInformation.empty
+        val newPipeline = oldPipeline.deepClone()
```

Comment:
I don't think this is correct. This means that the output row will contain all the input data and then some more, and that should not be true